### PR TITLE
rename configs dir to app_configs #82

### PR DESCRIPTION
**Pull request by goblinqueen**

### CODE REVIEW

File: shallow_backup.py [hunk 1]
```diff
@@ -356,11 +356,16 @@ def backup_all(dotfiles_path, packages_path, fonts_path, configs_path):
 	backup_configs(configs_path)
 
 
-def reinstall_config_files(configs_path):
+def reinstall_config_files(configs_path, old_configs_path):
 	"""
 	Reinstall all configs from the backup.
+	Look for old /configs/ dir for backward compatibility.
```
Comment (alichtman) [inline shallow_backup.py]: Nice catch.

**Comment by goblinqueen** [comment]
ok, removed the check

**Comment by alichtman** [comment]
In thinking about this further, I think it actually makes more sense to keep the directory named `configs` and back up `dots` inside of that folder, as opposed to separating `app_configs` and `dotfile_configs`.

I'll leave this open until I'm sure, but I don't know if this is the route I'll end up going.

**Comment by alichtman** [comment]
Yeah, follow up discussion in #86

**Comment by alichtman** [comment]
Thanks for putting this together. Sorry it wasn't merged in :(

**Pull request closed without merging**